utils/check-dependencies-exist.js

```diff
@@ -145,6 +145,13 @@ function formatErrorMessage(pkgFile, errors) {
 const modulePath = process.cwd();
 const packageName = require(path.join(modulePath, 'package.json')).name;
 
+// TODO: Figure out why we need this.
+if (modulePath.endsWith('/modules/react')) {
+  depCheckOptions.ignoreMatches.push('@workday/canvas-kit-react');
+} else if (modulePath.endsWith('/modules/labs-react')) {
+  depCheckOptions.ignoreMatches.push('@workday/canvas-kit-labs-react');
+}
```

Comment:
I'm not quite sure why this is needed. Oddly enough, `@workday/canvas-kit-react` only seems to complain about `@workday/canvas-kit-react/core` imports (i.e. `@workday/canvas-kit-react/icon` works fine), but `@workday/canvas-kit-labs-react` complains about any internal imports from `@workday/canvas-kit-labs-react` (e.g. `@workday/canvas-kit-labs-react/menu` or `@workday/canvas-kit-labs-react-color-picker`)